Makefile

```diff
@@ -36,12 +41,12 @@ test-skopeo:
 		rm -rf $${skopeo_path}
 
 validate: lint
-	@go vet ./...
-	@test -z "$$(gofmt -s -l . | tee /dev/stderr)"
+	@go vet $(PACKAGES)
+	@test -z "$$(gofmt -s -l . | grep -ve '^vendor' | tee /dev/stderr)"
 
 lint:
-	@out="$$(golint ./...)"; \
-	if [ -n "$$(golint ./...)" ]; then \
+	@out="$$(golint $(PACKAGES))"; \
+	if [ -n "$$(golint $(PACKAGES))" ]; then \
```

Comment:
Wait, we are running golint twice here? Please replace the second reference with `$$out`, perhaps as a separate commit.